packages/app/pages/course/[cid]/queue/[qid].tsx

```diff
@@ -28,7 +29,7 @@ export default function Queue() {
   const profile = useProfile();
   const router = useRouter();
   const { cid, qid } = router.query;
-  const role = profile?.courses.find((e) => e.course.id === Number(cid)).role;
+  const role = useRoleInCourse(Number(cid));
```

Comment:
Do we want to have `useRoleInCourse` hook do the casting of the `cid` from a string to a number so that we don't have to remember to cast it each time we call the function?